src/theme/components/subnav.ts

```diff
@@ -0,0 +1,207 @@
+  lineHeight: "1.5 !important",
+  position: "relative",
+  px: "s",
+  py: "xxs",
+  textDecoration: "none !important",
+  transition: "background-color 0.2s, color 0.2s !important",
+});
+
+const SubNav = subNavDefineMultiStyleConfig({
+  baseStyle: subNavDefinePartsStyle(
+    ({ backgroundColor, highlightColor }: SubNavStyleProps) => {
+      const defaultLabelColor = "ui.typography.body";
+      const highlightOrDefaultColor = highlightColor
+        ? `${highlightColor} !important`
+        : `${defaultLabelColor} !important`;
+      const highlightOrLinkColor = highlightColor
+        ? `${highlightColor} !important`
+        : "ui.link.primary !important";
+      const highlightOrBorderColor = highlightColor
+        ? `${highlightColor}`
+        : "ui.border.default";
+      const finalBackgroundColor = backgroundColor
+        ? backgroundColor
+        : "ui.link.primary-05 !important";
+      const primaryActionsStyles = {
+        ...commonStyles(),
+        svg: {
+          fill: defaultLabelColor,
+          margin: { base: "0", md: null },
+          _dark: {
+            fill: "ui.white",
+          },
+        },
+        _hover: {
+          backgroundColor: finalBackgroundColor,
+          color: highlightOrDefaultColor,
+          svg: {
+            fill: highlightOrDefaultColor,
+            _dark: {
+              fill:
+                backgroundColor !== undefined
+                  ? `${backgroundColor} `
+                  : "ui.white",
+            },
+          },
+        },
+      };
+      const secondaryActionsStyles = {
+        ...commonStyles(),
+        color: highlightOrLinkColor,
+        svg: {
+          fill: highlightOrLinkColor,
+          margin: { base: "0", md: null },
+          _dark: {
+            fill: backgroundColor
+              ? `${backgroundColor} !important`
+              : "dark.ui.link.primary-05 !important",
+          },
+        },
+        _hover: {
+          background: finalBackgroundColor,
+          svg: {
+            fill: highlightOrLinkColor,
+            _dark: {
+              fill: backgroundColor
+                ? `${backgroundColor}`
+                : "dark.ui.link.primary-05",
+            },
+          },
+        },
+      };
+      return {
+        base: {
+          ".selectedItem": {
+            color: highlightOrLinkColor,
+            fontWeight: "bold",
+            backgroundColor: finalBackgroundColor,
+            "&:hover": {
+              color: highlightOrLinkColor,
+            },
+          },
+          borderBottom: "1px solid",
+          borderColor: highlightOrBorderColor,
+          display: "flex",
+          justifyContent: "center",
+        },
+        container: {
+          maxWidth: "1280px",
+          // px: "s",
+          px: { base: "0", md: "xs" },
+          width: "100%",
+        },
+        listCleanup: {
+          // p: "xs",
+          p: { base: "s", md: "xs" },
+          gap: "xs",
```

Comment:
`gap` is not a rule for the `ul` element so you can remove this

Reply:
@EdwinGuzman The `gap` attribute is not a native attribute for `list`, but `display: flex` has been applied to the `list` element. Do you know if there are semantic implications when mixing these attributes?